src/handlers/lib/utility.js

```diff
@@ -465,9 +466,10 @@ const produceParticipantMessage = async (participantName, functionality, action,
     functionalityMapped = Enum.topicMap[functionality][action].functionality
     actionMapped = Enum.topicMap[functionality][action].action
   }
-  return await Kafka.Producer.produceMessage(updateMessageProtocolMetadata(message, functionality, action, state),
+  let result = await Kafka.Producer.produceMessage(updateMessageProtocolMetadata(message, functionality, action, state),
```

Comment:
Why is the result assigned to a variable and then returned? Could it not be done in one line with return await?